### DISCUSSION TITLE
Mojolicious's official blog

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [mojolicious/mojo#1727] [Ideas]
Have Mojolicious core team ever thought about Mojolicious's official blog?

This is a blog that provides useful information about Mojolicious for general users(Business users, Lite users, Application users).

**Comment by kraih** [comment]
Yes, the main problem is that almost nobody on the team wants to blog anymore. I've been thinking about forwarding blogs.mojolicious.org to the [dev.to topic](https://dev.to/t/mojolicious) though, to encourage more blogging in the community.

**Reply by kraih** [reply]
I should probably mention that at some point we did have an official team blog for like 2 years, and nobody used it, instead everyone just ended up blogging on their personal blogs. The project [Twitter account](https://twitter.com/perlmojo) has been working out a little better and we'll probably keep using that for announcements.

**Reply by kraih** [reply]
I have activated [blogs.mojolicious.org](http://blogs.mojolicious.org) now. So everyone in the community can contribute just by tagging their dev.to posts with the `#mojolicious` tag.

**Reply by yuki-kimoto** [reply]
I understand that Mojolicious team use Twitter for announcements, and blog become personal things.

I write only one about mojo.maybe.

Initially Mojolicious::Plugin::AutoRoute was written before render_maybe was implemented .

There is no intention of intentionally using something private, it remains the same.

It will be replaced at some future.

**Reply by kraih** [reply]
I've also started blogging a bit on dev.to. https://dev.to/kraih

**Comment by MartinMcGrath** [comment]
blogs.mojolicious.org sounds interesting, perhaps something similar to perl.com
would work, where people submit posts as PRs for approval before they are
published.

**Reply by kraih** [reply]
GitHub discussions can't handle email answers yet. This now looks like an answer to the original post, not mine.

**Reply by MartinMcGrath** [reply]
> GitHub discussions can't handle email answers yet. This now looks like an answer to the original post, not mine.

Good to know, thanks.

**Comment by yuki-kimoto** [comment]
I start to write Perl and Mojolicious content on Dev community!
